linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_profile/ansible/shared.yml

```diff
@@ -5,24 +5,37 @@
 # disruption = low
 {{{ ansible_instantiate_variables("var_accounts_user_umask") }}}
 
-- name: Check if umask is already set
+- name: '{{{ rule_title }}} - Locate profile configuration files where umask is defined'
+  ansible.builtin.find:
+    paths:
+      - /etc/profile.d
+    patterns:
+      - sh.local
+      - '*.sh'
+    contains: ^[\s]*umask\s+\d+
+  register: result_profile_d_files
+
+- name: '{{{ rule_title }}} - Replace existing umask value in files from /etc/profile.d'
```

Comment:
Per the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible) this should be in title case.

Reply:
fixed